npm/packages/trial/src/graph.ts

```diff
@@ -840,4 +854,11 @@ class TrialGraph {
   private _graphId(): string {
     return "trial-graph-" + this.graphId;
   }
+
+  //dataflow:
+
+  generateDataflow() {
+    return 1;
```

Comment:
Check what does it do

Reply:
This should also be excluded. It was a first attempt to implement the file access, but this was in the wrong file.